scalding-parquet-scrooge-cascading/src/main/java/com/twitter/scalding/parquet/scrooge/cascading/ParquetScroogeScheme.java

```diff
@@ -16,13 +16,13 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package com.twitter.scalding.parquet.scrooge;
+package com.twitter.scalding.parquet.cascading.scrooge;
```

Comment:
@piyushnarang looks like the package name does not match the file name. not scrooge/cascading, but we have cascading.scrooge.

Reply:
I can push a follow up fix for this tomorrow.